test/Parse/multiline_string.swift

```diff
@@ -109,6 +109,31 @@ _ = """
 // CHECK: "Twelve\nNu"
 
 _ = """
+  newline \
+  elided
+  """
+
+// CHECK: "newline elided"
+
+// contains trailing whitepsace
+_ = """
+  trailing \
+  \("""
+    substring1 \
+    \("""
+      substring2 \          
+      substring3
+      """)\
```

Comment:
This looks like a mistake in the updated proposal. It should read “immediately preceding”.